src/openzaak/components/zaken/models/zaken.py

```diff
@@ -805,7 +805,10 @@ class Rol(ETagMixin, APIMixin, models.Model):
         editable=False,
         db_index=True,
     )
-    roltoelichting = models.TextField(max_length=1000)
+    roltoelichting = models.TextField(
+        max_length=1000,
+        blank=True,
```

Comment:
As @annashamray suggested, let's mention in the help text that this deviates from the standard
```suggestion
        blank=True,
        help_text=mark_experimental(_("Toelichting bij de rol (dit veld wijkt af van de standaard, omdat een lege string `\"\"` is toegestaan).")),

```